cloud-config/aws/README.md

```diff
@@ -0,0 +1,156 @@
+This Cloud-Config setup is intended to be used with the Ubuntu Server 18.04 LTS AMI in AWS. You can search for the latest AMI id for your AWS zone at https://cloud-images.ubuntu.com/locator/ec2. Search for `18.04 amd64 hvm:ebs-ssd`
```

Comment:
I think you can drop the bit about latest AMI ID. I don't think anyone should care.